<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ventana Mecanico</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Especialidad">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Introduce la especialidad</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="FechaContratacion">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVolver">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
